lib/MentionWrapper.js

```diff
@@ -206,10 +206,17 @@ var MentionWrapper = function (_Component) {
       var tokens = textBeforeCaret.split(/\s/);
       var lastToken = tokens[tokens.length - 1];
 
-      // check if the text befor the caret ends with the last word
-      var triggerIdx = textBeforeCaret.endsWith(lastToken) ? textBeforeCaret.length - lastToken.length : -1;
+      // Get active trigger
+      var activeTrigger = this.triggers.find(function (trigger) {
+        return lastToken.includes(trigger);
```

Comment:
don't forget to compile and push

Reply:
That is the auto output of my compiler? I never modified that file directly.

Reply:
nvm, i'm dumb